vignettes/quickstart.Rmd

```diff
@@ -88,54 +67,60 @@ function include:
 
 - `data` the primary longitudinal data.frame containing the outcome variable and all covariates
 - `data_ice` a data.frame specifying which visit (if any) the patient's intercurrent 
-event occurred on. If the patient had multiple intercurrent events (ICE) this should 
-specify the date on which the first non-MAR intercurrent event occurred on. It also 
+event (ICE) occurred on, or more precisely the first visit in which the outcome has been affected by the ICE. If the patient had multiple ICEs this should 
+specify the first visit affected by the ICE addressed with a non-MAR imputation strategy. It also 
```

Comment:
Perhaps:

"first visit affected by the ICE addressed with a non-MAR" -> "first visit affected by the ICE which is to be addressed by a non-MAR"
 
 Perhaps

Reply:
done